### DISCUSSION TITLE
「如果今天面試官是機器人」

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#974] [General]
「如果今天面試官是機器人」
我們從申請大學入學開始了我們面試的第一扇門
而當面試官由真人轉變成ai後，透過觀察面試者臉部的微表情與應答進行評分。
然而這對於平時溝通較無情緒起伏或臉部表情癱瘓的人來說是非常不利的
![AAFBC619-6B16-4144-9F6C-14AB0CBFF8FB](https://user-images.githubusercontent.com/113414274/210713918-a921faf8-f5e6-469e-b06c-90663881aea0.jpeg)
呢。
https://www.cheers.com.tw/article/article.action?id=5096862&page=3

**Comment by Doreen331** [comment]
人類目前贏過機器人最大的地方就是我們有情緒，若面試這類需要情緒感知能力的工作交給AI，感覺有點矛盾

**Comment by dinosaur1009** [comment]
我認為面試很大一個部分是臨場的判斷還有情緒反應的管理，而AI只從臉部表情判斷，有點過於狹隘

**Comment by ruthchen1020** [comment]
機器人雖然判斷的很準，但人跟人相處起來的感覺不能用算的吧。

**Comment by peggy0329** [comment]
面試最重要的一點就是考驗面試者的臨場反應
如果只變成用AI判斷臉部表情的話，好像無法精準的了解面試者的靈機應變能力
而且我認為實際與人交流，才能道出溫度
有時候面試真的是看與面試官合不合的感覺

**Comment by AnAnistired** [comment]
我覺得面試官是機器人反而能讓每個面試者獲得更公平公正的結果，先不論人類面試官的主觀想法和專業程度，若是一整天的面試行程，誰能保證面試官在面對最後一個面試者時能和第一個面試者時有一樣的耐心和判斷標準？影響人類的因素還是太多了⋯⋯

**Comment by manipulateed** [comment]
我覺得各有利弊，以人當考官能更加明白考生所帶出來的情感思想與談吐是否舒服，當然這個非常主觀，所以AI也可以避免這些問提，酖這不免會導致面試過程太公式化而不能找出需要的確切人才。

**Comment by swerfg03joe** [comment]
面試如果只透過AI我認為是非常危險的，因為AI會被設定對某幾項為評分依據，那麼如果來面試的人剛好這部分特別突出其他能力低下那就沒辦法過濾出來，所以面試應該要採取人工為主這樣才能測出臨機應變的能力，然後AI作為輔助測試

**Comment by HuangHsinTzu** [comment]
我覺得最適當的做法是以真人面試官為主，AI面試官為輔
我不認為一個人的好壞可以用一個general的演算法去做評斷
而且萬一AI的演算法與喜好被摸透可能會減少人性多樣性
真人面試官也才會有面試(可怕)的感覺，可以考驗受試者的抗壓性
但在微表情觀察的部分AI的確有一定的參考價值

**Comment by delia07152** [comment]
雖然從一個人的表情可以多少判斷出這個人的性格，但我覺得才能方面是透過言語表達出來的，以及他的資料內容，性格跟才能一樣重要，但既然人類兩個都可以自行判斷且做得比ai好或是更能讓面試者信服的話，那我覺得還是由人類來執行這項工作比較好。

**Comment by ching020721** [comment]
人工判定可能比較主觀，但能結合綜合能力考量，若能將這個AI面試官當作其中一個評分標準(可能占比總分一部份)，使得人類面試官的主觀判定不再是唯一準則的話，可以更好的選擇公司渴望的人才，那也是很好的發展方向。

**Comment by Sylvie0601** [comment]
我覺得用機器人可以綜合評判，但我覺得有時候人的情緒表達那一些細微的人的表現感覺還是需要人做評判，不過我覺得機器人可以做一些基本評判也不錯，可以減少面試官的負擔，若是一次要很了解這個人是否符合要求，感覺可以先經過機器人評斷，再從人判斷感覺也不錯。

**Comment by Kotori0501** [comment]
利用AI來臉部辨識一個的面部表情雖然很方便，但我覺得利用AI來判斷一個人的表情比較偏向輔助耶，因為人與人間的溝通，通常會比AI更快速的了解一個人的個性，而且只光靠測試面部表情，我覺得是沒什麼用的，因為面試時有些人會把緊張寫在臉上，這樣就比較難知道對方的個性，所以臉部辨識還是用於輔助會比較好

**Comment by Tuoresa** [comment]
我認為大量統計數據能應用在眾多方面，並提升判斷的精準度與效率。

對於「面試」這種是用來增進人與人的實際接觸並真實感受彼此的機制，
若AI進行輔助分析以幫助面試官更有方向性的去透過真實對談應證結果，可能會帶來正面影響
但要是直接取代，我認為結果必會偏剖也有點本末倒置，
儘管科技再如何發達，人與人的真實交流仍是無可取代的

**Comment by bonny123865** [comment]
AI在提升生產力跟提高準確度上的確帶來很多進步
但是人的能力或是工作上的表現可能會被面試時的緊張給蓋過去
而且AI面試的標準可能難以評估好與壞

**Comment by dabbie109303501** [comment]
目前對AI能"共情"的說法還無法證實
那麼還能相信AI能正確判斷人類的面部表情和喜努哀樂嗎
若因AI判斷失誤而錯失心心念念的學校 又要如何對受試者負責?

**Comment by wenchi9023** [comment]
對於一個不講話臉看起來很兇的我來說，AI這項技術對我真不利 : (
而且為了通過AI面試，人人都有可能會隱藏自己的真實性格，如此一來該技術對公司還會有幫助嗎

**Comment by Jzd314** [comment]
AI面試只看的到表面的情緒，對於內在或鏡頭外的表現無法進行判斷，AI考官感覺更適合用來進行初步篩選，由真人來進行最後面試會比較好

**Comment by ghost** [comment]
我覺得這篇文章太簡略了，是如何運作的都沒說明。不過就就面部表情來做為分析標準的話，我認為這只會招聘到一些浮誇的演員，我認為對於實際內容去做詢問，並分析其結果的合理性與公司理念的符合性或許比較實際

**Comment by orianna901001** [comment]
就共情來說，面試官如果是機器人的話，在技術層面或許可以挑出最最最優秀的人才，但面試可能看的不單單只是個人技術，我認為甚至還包含了說話藝術、創意度、個人特質等特色。

**Comment by harrycoconut** [comment]
我認為這其實ai面試官是個非常不錯的想法！若是你在面試過程中騙人的話，ai馬上就能辨識出來，同時也能訓練我們在緊張的壓力環境下如何保持鎮定，並且很好的控制面部表情。對我而言，ai面試官絕對是個我們未來必定面對的挑戰！

**Comment by Chienyuuu6** [comment]
我不怎麼支持AI面試官ㄉidea耶，我認為面試這件事情為的就是實質上人與人的交流來判斷面試者與雇主之間的合適度。

人類的情感真的很複雜，某些直覺更不可能是能夠訓練給機器的，如果仰賴AI面試官我覺得這世界的失業率可能也會提升⋯

**Comment by ALAN30359704** [comment]
個人認為要把情緒量化是個大問題，若有如何轉成可計算的資料來判斷我覺得更好。

**Comment by KevinCheng42** [comment]
我覺得AI或許還不能代替面試官進行面試，但它能夠輔助面試官捕捉面試者的表情和談話語氣，以分析面試者的人格特質或性格，幫助面試官在篩選時多一份依據，選擇公司需要的人才～

**Comment by ziihui** [comment]
非常同意你所提出的觀點！機器人面試官的出現確實對於一些人來說可能會帶來不公平的評估。如果面試者平時情緒表現較為克制或表情較為呆板，那麼面試官可能會對其進行不公正的評估。因此，在機器人面試官的面前，我們或許需要更加注重言語表達和思維邏輯等方面，這樣才能夠更好地展現自己的實力。當然，這也提醒著我們在日常生活中要多加關注和學習情緒表達和人際溝通的技巧，這樣才能夠更好地應對各種面試和人際交往的場合。

**Comment by Jamesmoneya** [comment]
如果他要透過AI來代替真人的面試，我覺得他的資料庫會變得非常重要，而且即使有強大的資料庫，有時候去判斷一個人，是她散發的氣質，他的氣場，他給人的感覺，我不覺得這種感覺可以量化成未AI判斷的依據

**Comment by hardtagottaname** [comment]
我認為用AI頂替真人面試除了對於被面試者極不尊重外，也讓公司損失能夠優先了解職場共事者的機會。用AI面試讓被面試者無法和真人互動，對我來說感覺被敷衍。除此之外，面試者的用詞、態度等等微表情有可能會因為誤差被AI扭曲，讓被面試者喪失資格，讓我覺得不公平。

**Comment by CCello** [comment]
我認為機器人面試官可能更加嚴格、公正和客觀，但卻無法像真人面試官一樣有人情味和靈活度，因此可能無法瞭解到面試者的真實狀況和能力，進而影響評分和錄取結果。所以，如果今天面試官是機器人，我們需要更加認真和專注地面對面試，同時也需要思考如何更好地展現自己的能力和潛力，以獲得更好的結果。

**Comment by ashleyy-huang** [comment]
我覺得透過機器人來面試並不是一個很好的選擇，因為機器人沒辦法像人類一樣透過感官來認識一個人，它會容易忽略到每個人的特質和小細節，我認為如果一間公司的應徵面試需要透過機器人來執行的話對於員工是沒有誠意的
